client/state/comments/actions.js

```diff
@@ -160,6 +161,11 @@ export function requestPostComments( siteId, postId, status = 'approved' ) {
 	};
 }
 
+export const requestCommentsList = query => ( {
+	type: COMMENTS_LIST_REQUEST,
```

Comment:
Dispatching from console I don't see the initial `COMMENTS_LIST_REQUEST` but I do see the other dispatched events:
![screen shot 2017-05-25 at 4 11 28 pm](https://cloud.githubusercontent.com/assets/1270189/26474266/d7fe5cd4-4164-11e7-946a-24f616220c15.png)